.github/workflows/ci-full.yml

```
@@ -0,0 +1,2790 @@
+        uses: actions/cache@v4
+        id: pg-cache
+        with:
+          path: ~/.spindb/bin
+          key: spindb-pg-18-v2-${{ runner.os }}-${{ runner.arch }}
+
+      - name: Install PostgreSQL via SpinDB
+        run: pnpm start engines download postgresql 18
+
+      - name: Create container with seed data
+        shell: bash
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2034:warning:4:1: i appears unused. Verify use (or export if used externally)

(shellcheck)